packaging/win32/installer.nsi

```diff
@@ -65,8 +65,9 @@ SetCompressorDictSize 64
 !define MUI_FINISHPAGE_SHOWREADME_NOTCHECKED
 !define MUI_FINISHPAGE_SHOWREADME_TEXT "Create Desktop Shortcut"
 !define MUI_FINISHPAGE_SHOWREADME_FUNCTION CreateDesktopShortcut
-# Run Parsec after install
-!define MUI_FINISHPAGE_RUN "$INSTDIR\parsec.exe"
+# Run Parsec after install, use explorer.exe but un-elevate priviledges
```

Comment:
Can you improve the comment, for me it's not obvious that we are using explorer.exe to avoid running parsec as admin (or maybe add a link to this PR)